src/ApiService/ApiService/onefuzzlib/Defs.cs

```diff
@@ -39,8 +39,131 @@ public static class Defs {
                 )},
             MonitorQueue: ContainerType.ReadonlyInputs)
     },
+    {
+        TaskType.DotnetCoverage, new TaskDefinition(
```

Comment:
`DotnetCoverage` task definition was already present:
https://github.com/microsoft/onefuzz/blob/ac54c576aef5c444cb85d7f6420769c35905b24d/src/ApiService/ApiService/onefuzzlib/Defs.cs#L558

Reply:
Same for python:
https://github.com/microsoft/onefuzz/blob/ac54c576aef5c444cb85d7f6420769c35905b24d/src/api-service/__app__/onefuzzlib/tasks/defs.py#L561

Reply:
Nice catch, thanks! I did originally _move_ it, but lost that commit at some point 😅.

Reply:
Looks like I _had_ gotten the Python dupe, but not the C# one.